services/src/test-library/src/main/java/org/openkilda/testing/service/elastic/ElasticServiceImpl.java

```diff
@@ -0,0 +1,53 @@
+/* Copyright 2018 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.testing.service.elastic;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+/**
+ * An interface to the Elastic Search. Set following variables in the kilda.properties to work:
+ * elasticsearch.endpoint = URL to Elastic
+ * elasticsearch.username = User with read access
+ * elasticsearch.password = User's password
+ */
+
+@Component
+public class ElasticServiceImpl implements ElasticService {
```

Comment:
This whole class is pointless as for me. I can basically pass any uri and any httpentity to the 'getLogs' method, it has nothing to do with actual logs and don't even restrict me to some 'logs' endpoint. Most of the code should move from groovy helper to here, or just remove this class and autowire the resttemplate in helper.